### PR TITLE
Fix test positive install modular errata

### PR DESCRIPTION
**Pull request by swadeley**
Moved setup to before session start
  Fixed fake package name conflict

test setup was taking so long that on exit, context manager clean up, selenium session had expired.
Moving setup to before session start revealed the real cause of the test fail. Hat tip to @tpapaioa for that insight.

### CODE REVIEW

**Comment by swadeley** [comment]
robottelo]$ pytest --show-capture=no tests/foreman/ui/test_hostcollection.py::test_positive_install_modular_errata
========================= test session starts =================================================
platform linux -- Python 3.9.5, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /sat6/robottelo, configfile: pyproject.toml
plugins: xdist-2.3.0, mock-3.6.1, ibutsu-1.16, forked-1.3.0, services-2.2.1, cov-2.12.1, reportportal-5.0.8
collected 1 item                                                                                                                                                                                  

tests/foreman/ui/test_hostcollection.py .                                                                                                                                                   [100%]

===================================== 1 passed, 3 warnings in 1169.93s (0:19:29) ====================

File: tests/foreman/ui/test_hostcollection.py [hunk 1]
```diff
@@ -713,18 +713,14 @@ def test_positive_install_modular_errata(
 
     :CaseLevel: System
     """
+    stream = "0"
+    version = "20180704111719"
+    _module_install_command = 'dnf -y module install {}:{}:{}'.format(
```
Comment (mshriver) [inline tests/foreman/ui/test_hostcollection.py:718]: Please use f-strings here

**Review by mshriver — APPROVED** [review]
Approved with some f-string use

File: tests/foreman/ui/test_hostcollection.py (hunk 1 above)
Comment (swadeley) [reply tests/foreman/ui/test_hostcollection.py]: Fixed, testing in progress.

**Comment by swadeley** [comment]
robottelo]$ pytest --show-capture=no tests/foreman/ui/test_hostcollection.py::test_positive_install_modular_errata
============================== test session starts =================================================
platform linux -- Python 3.9.5, pytest-6.2.4, py-1.10.0, pluggy-0.13.1
shared_function enabled - OFF - scope:  - storage: file
rootdir: /sat6/robottelo, configfile: pyproject.toml
plugins: xdist-2.3.0, mock-3.6.1, ibutsu-1.16, forked-1.3.0, services-2.2.1, cov-2.12.1, reportportal-5.0.8
collected 1 item                                                                                                                                                                                  

tests/foreman/ui/test_hostcollection.py .                                                                                                                                                   [100%]

============================== 1 passed, 3 warnings in 1055.77s (0:17:35) =================================

**Comment by swadeley** [comment]
Thank you

**Pull request merged**